kivy/core/window/window_sdl2.py

```diff
@@ -714,7 +714,8 @@ def request_keyboard(self, callback, target, input_type='text'):
 
     def release_keyboard(self, *largs):
         super(WindowSDL, self).release_keyboard(*largs)
-        self._win.hide_keyboard()
+        if self._system_keyboard.widget:
+            self._win.hide_keyboard()
         self._sdl_keyboard = None
```

Comment:
I'm not sure why this would work. `widget` seems to be a private property of the keyboard. Why would we only release the keyboard when there's a virtual keyboard?